### DISCUSSION TITLE
Accessibility: Toggle Group used as 'radio' group?

### DISCUSSION THREAD

**Discussion by DPCamino** [radix-ui/website#552] [Q&A] [answered]
Is it acceptable, in terms of accessibility, to use the Toggle Group like a radio option if it has same characteristics as a radio group (e.g., default value selected and only one can be selected at a time)?

Both use roving tabindex.

Label for this group would be something like "please select one of following" and each option is a binary on/off, and only one can be selected, and there would still be an icon/background color change to indicate selected - that appears to be be practically similar but wasn't sure if there is something that makes this unacceptable.

Thanks!

**Comment by benoitgrelard** [comment, marked as answer]
Hey @SemperYak, in `type="single"` mode, yes it should be acceptable accessibility-wise.
The biggest difference between a `RadioGroup` and a `ToggleGroup` with `type="single"` will be the fact that `ToggleGroup` is less suited for forms as it won't bubble a value change. So it kinda depends how you're using it.

**Reply by DPCamino** [reply]
I have a couple use cases and yes, one of them is a form. If that's the case I will then use the appropriate `RadioGroup` and style it to match what I have with my existing `ToggleGroup`

Thank you @benoitgrelard!
